src/main/java/gov/cabinetoffice/gap/adminbackend/services/SchemeService.java

```diff
@@ -169,4 +169,40 @@ public void patchCreatedBy(GrantAdmin grantAdmin, Integer schemeId) {
         scheme.setFunderId(grantAdmin.getFunder().getId());
         this.schemeRepo.save(scheme);
     }
+
+    public List<SchemeDTO> getPaginatedOwnedSchemes(Pageable pagination) {
+        final AdminSession adminSession = HelperUtils.getAdminSessionForAuthenticatedUser();
+        final int grantAdminId = adminSession.getGrantAdminId();
+        final List<SchemeEntity> schemes = this.schemeRepo
+                    .findByCreatedByOrderByCreatedDateDesc(grantAdminId, pagination);
+
+        return this.schemeMapper.schemeEntityListtoDtoList(schemes);
+    }
+
+    public List<SchemeDTO> getOwnedSchemes() {
+        final AdminSession adminSession = HelperUtils.getAdminSessionForAuthenticatedUser();
+        final int grantAdminId = adminSession.getGrantAdminId();
+        final List<SchemeEntity> schemes = this.schemeRepo
+                .findByCreatedByOrderByCreatedDateDesc(grantAdminId);
+
+        return this.schemeMapper.schemeEntityListtoDtoList(schemes);
+    }
+
+    public List<SchemeDTO> getPaginatedEditableSchemes(Pageable pagination) {
+        final AdminSession adminSession = HelperUtils.getAdminSessionForAuthenticatedUser();
+        final int grantAdminId = adminSession.getGrantAdminId();
+        final List<SchemeEntity> schemes = this.schemeRepo
+                .findByCreatedByNotAndGrantAdminsIdOrderByCreatedDateDesc(grantAdminId, grantAdminId, pagination);
+
+        return this.schemeMapper.schemeEntityListtoDtoList(schemes);
+    }
+
+    public List<SchemeDTO> getEditableSchemes() {
+        final AdminSession adminSession = HelperUtils.getAdminSessionForAuthenticatedUser();
```

Comment:
Thinking it might be better to grab the admins session in the controller layer and then pass the admin id into all of these methods? means we only have to fetch it once

Reply:
yeah good call mate, will do that just now

Reply:
sorted